analyzer.go

```diff
@@ -56,6 +56,9 @@ func (a *Analyzer) analyzeLayers(appMeta LayersMetadata, cache Cache) error {
 	var cacheMeta CacheMetadata
 	if cache != nil {
 		var err error
+		if !cache.Exists() {
+			a.Logger.Info("cache does not exist")
```

Comment:
I still think we should tweak this slightly. `does not exist` seems potentially misleading given this could happen if we do not have the correct auth for example. Or maybe the cache volume directory exists but doesn't have metadata/layers in it. Maybe:
```suggestion
			a.Logger.Info("Layer cache not found")
```